CMakeLists.txt

```diff
@@ -39,3 +41,10 @@ endif()
 if (mmtf_build_examples)
     add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/examples)
 endif()
+
+if (mmtf_install_system)
+    install(
+        DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/
+        DESTINATION  "/include"
```

Comment:
I think the leading slash is wrong, this must be a relative path (see https://cmake.org/cmake/help/v3.0/command/install.html )

Reply:
The destination it actually installs to is `${DCMAKE_INSTALL_PREFIX}/include` and `DCMAKE_INSTALL_PREFIX` defaults to `/usr/local/`

It is slightly wrong in that you end up with `/usr/local//include/` if you have a / at the end of your `DCMAKE_INSTALL_PREFIX`, but that's not really a problem.  but it's safer this way if you want to set your `DCMAKE_INSTALL_PREFIX` manually and you forget the trailing slash.

Reply:
I'm confused, because it doesn't work for me.
```
cmake -DCMAKE_INSTALL_PREFIX=/tmp/prefix ..
make install
```
```
-- Installing: /include
CMake Error at cmake_install.cmake:49 (file):
  file INSTALL cannot make directory "/include": Permission denied
```
After removing the slash it's correct:
```
-- Installing: /tmp/prefix/include
-- Installing: /tmp/prefix/include/mmtf.hpp
...
```
And I'm repeating myself, but the cmake docs say: " If a full path (with a leading slash or drive letter) is given it is used directly."

Reply:
I must have some strange setting or environment set, works fine for me. Although it's not reproducible in docker. I'll switch it.